### DISCUSSION TITLE
About Wazuh

### DISCUSSION THREAD

**Discussion by zbalkan** [CISOfy/lynis#1321] [Q&A]
Dear Lynis maintainers and contributors, 

Recently, I created a couple of [issues](https://github.com/CISOfy/lynis/issues/created_by/zbalkan), and [PRs](https://github.com/CISOfy/lynis/pulls/zbalkan) related to Wazuh. For those who do not know about Wazuh, I highly recommend having a look at the documentation, especially [compliance](https://documentation.wazuh.com/current/compliance.html) pages. It has a wide coverage of several compliance requirements and a huge intersection with Lynis tests. Even though I tried to be as objective as possible, I might be biased as a satisfied user and miss crucial details.

Can you please give a feedback on the suggestions? 

Best regards, 

Zafer

**Comment by konstruktoid** [comment]
I see no issues with your PRs.

**Reply by zbalkan** [reply]
Thank you for the comment. Now I'm glad that I made a meaningful contribution.
